cime_config/machines/config_machines.xml

```diff
@@ -4259,7 +4259,7 @@ AMD EPYC 7713 64-Core (Milan) (256GB) and 4 nvidia A100'</DESC>
       <arguments>
         <arg name="label"> --label</arg>
         <arg name="num_tasks"> -n {{ total_tasks }} -N {{ num_nodes }}</arg>
-        <arg name="thread_count">-c $SHELL{echo 256/`./xmlquery --value MAX_MPITASKS_PER_NODE`|bc}</arg>
+        <arg name="thread_count">-c $ENV{OMP_NUM_THREADS}</arg>
```

Comment:
This now works, and I can run on gpus on chicoma. The commands were:
```
salloc -N 1 -t 2:0:0 --account=g23_nonhydro_g --partition=gpu
cd /usr/projects/climate/mpeterse/repos/E3SM/pr/cime/scripts
./create_test SMS.T62_oQU120_ais20.MPAS_LISIO_TEST.chicoma-gpu_gnugpu \
  --walltime 1:00:00 --wait \
  -p g23_nonhydro_g
```
The command now runs with `srun -c 1`, but I chose to use `-c $ENV{OMP_NUM_THREADS}` based on @philipwjones' comment above that it is the number of threads. About half the machines in `config_machines.xml` use this convention. 

@philipwjones do you agree with this change?

Reply:
Yep, I agree.